crates/matrix-sdk-crypto/src/verification/machine.rs

```diff
@@ -128,21 +124,8 @@ impl VerificationMachine {
         &self,
         device: ReadOnlyDevice,
     ) -> Result<(Sas, OutgoingVerificationRequest), CryptoStoreError> {
-        let identity = self.store.get_user_identity(device.user_id()).await?;
-        let own_identity =
-            self.store.get_user_identity(self.own_user_id()).await?.and_then(|i| i.into_own());
-        let private_identity = self.private_identity.lock().await.clone();
-
-        let (sas, content) = Sas::start(
-            private_identity,
-            device.clone(),
-            self.store.clone(),
-            own_identity,
-            identity,
-            None,
-            true,
-            None,
-        );
+        let identities = self.store.get_identities(device.clone()).await?;
+        let (sas, content) = Sas::start(identities, None, true, None);
```

Comment:
It's not at all clear what `None, true, None` means from the call site. Would be nice to find a more readable solution.